Source/JavaScriptCore/bytecode/BytecodeList.rb

```diff
@@ -1878,6 +1878,15 @@
         useDefault: bool,
     }
 
+op :array_new_data,
+    args: {
+        dst: VirtualRegister,
+        size: VirtualRegister,
+        offset: VirtualRegister,
+        typeIndex: unsigned,
+        dataIndex: unsigned,
+    }
```

Comment:
We should implement these things as a builtin call instead of adding a new byte code. See call_builtin implementation.